broken-site/broken-site-impl/src/main/java/com/duckduckgo/brokensite/impl/BrokenSiteReportRepository.kt

```diff
@@ -59,15 +59,15 @@ interface BrokenSiteReportRepository {
 
     fun resetRefreshCount()
     fun addRefresh(url: Uri, localDateTime: LocalDateTime)
-    fun getAndUpdateUserRefreshesBetween(t1: LocalDateTime, t2: LocalDateTime): Int
+    fun checkForRefreshPatterns(currentDateTime: LocalDateTime): Set<Int>
```

Comment:
Would probably rename to `getRefreshPatterns` rather than check. Check doesn't tell me what is the thing that is checking so a bit confusing. Also, this returns the refresh patterns but they are not associated to any URL so how do you know it's relevant for the current site?

Reply:
The [in-memory store](https://github.com/duckduckgo/Android/pull/5862/files#diff-d47e78d418739db283d4d048f24012def22255385c67aea5844edb0852bf0998) is keeping track of these per url, so it's built into the underlying method called by the repository.